i18n/zh/docusaurus-plugin-content-docs/current/architecture/edge/devicetwin.md

```diff
@@ -2,353 +2,327 @@
    - dealTwinSync
 
-**dealTwinUpdate**: dealTwinUpdate() updates the device twin information for a particular device.
-- The devicetwin update message can either be received by edgehub module from the cloud or from
-the MQTT broker through the eventbus component (mapper will publish a message on the device twin update topic) .
-- The message is then sent to the device twin controller from where it is sent to the device twin module.
-- The twin module updates the twin value in the database and sends the update result message to the communication module.
-- The communication module will in turn send the publish message to the MQTT broker through the eventbus.
+**dealTwinUpdate**: dealTwinUpdate() 更新特定设备的设备孪生信息。
+- The devicetwin 更新消息可以由 Edgehub 模块从云端接收，也可以通过 eventbus 组件从 MQTT 代理接收（映射器将在 device twin 更新 topic 上发布消息）。
+- 然后，该消息被发送到设备孪生控制器，从那里它被发送到设备孪生模块。
+- Twin 模块更新数据库中的孪生值，并将更新结果消息发送至通信模块。
+- 通信模块将依次通过事件总线将发布消息发送到 MQTT 代理。
 
   ![Device Twin Update](/img/devicetwin/devicetwin-update.png)
 
 
-**dealTwinGet**: dealTwinGet() provides the device twin  information for a particular device.
-  - The eventbus component  receives the message that arrives on the subscribed twin get topic and forwards the message to devicetwin controller, which further sends the message to twin module.
-  - The twin module gets the devicetwin related information for the particular device and sends it to the communication module, it also handles errors that arise when the device is not found or if any internal problem occurs.
-  - The communication module sends the information to the eventbus component, which publishes the result on the topic specified .
+**dealTwinGet**: dealTwinGet() 提供特定设备的设备孪生信息。
+  - eventbus 组件接收到达订阅的 twin get 主题的消息，并将该消息转发到 devicetwin 控制器，devicetwin 控制器进一步将消息发送到 twin 模块。
+  - twin 模块获取特定设备的 devicetwin 相关信息并将其发送到通信模块，它还处理未找到设备或发生任何内部问题时出现的错误。
+  - 通信模块将信息发送到 eventbus 组件，eventbus 组件将结果发布到指定的主题上。
 
   ![Device Twin Get](/img/devicetwin/devicetwin-get.png)
 
 
-**dealTwinSync**: dealTwinSync() syncs the device twin information to the cloud.
-  - The eventbus module receives the message on the subscribed twin cloud sync topic .
-  - This message is then sent to the devicetwin controller from where it is sent to the twin module.
-  - The twin module then syncs the twin information present in the database and sends the synced twin results to the communication module.
-  - The communication module further sends the information to edgehub component which will in turn send the updates to the cloud through the websocket connection.
-  - This function also performs operations like publishing the updated twin details  document, delta of the device twin as well as the update result (in case there is some error) to a specified topic through the communication module,
-  which sends the data to edgehub, which will send it to eventbus which publishes on the MQTT broker.
+**dealTwinSync**: dealTwinSync() 将设备孪生信息同步到云端。
+  - eventbus 模块接收有关订阅的 Twin 同步主题的消息。
```

Comment:
`device twin update topic`,  `device twin get topic` and `device twin sync topic`  are best translated in the same form